<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmcrmpregled</class>
 <widget class="QDialog" name="frmcrmpregled">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Datum dolaska</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtDatumDolaska">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Kupac</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtKupac"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Prodavac</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtProdavac"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Unio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtUnio"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblRacunVeza">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Racun veza</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </item>
     <item>
      <widget class="QTextEdit" name="txtOpis"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnIzmijena">
         <property name="text">
          <string>Izmijena</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnIzlaz">
         <property name="text">
          <string>Izlaz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnIzlaz</sender>
   <signal>pressed()</signal>
   <receiver>frmcrmpregled</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
